cogs/utils/constants.py

```diff
@@ -2,6 +2,7 @@
 
 from discord import Color
 
+creator = "TheOriginalDude#0585"
```

Comment:
```suggestion
creator = "The-Codin-Hole team"
```
It seems a bit unfair to take all the credit, we are all working on it now.